theme/templates/base.html

```diff
@@ -25,6 +25,9 @@
     {# should be equivalent to rel=canonical -- used for aggregating likes #}
     <meta property="article:publisher" content="https://www.facebook.com/pages/Mixxx-DJ-Software/21723485212"/>
 
+    <!-- Mastodon verification backlink -->
+    <a rel="me" href="https://floss.social/@mixxx">Mastodon</a>
+
```

Comment:
Does this really have to go in the header? It creates an ugly artifact. Wouldn't it make more sense to add this as a separate entry in the Community links at the bottom? Or Just add `display: none` so it gets hidden. 
![Screenshot from 2023-01-05 22-35-12](https://user-images.githubusercontent.com/12380386/210884462-5f6767b6-c410-459f-bba1-05b1388de291.png)

Reply:
oh sorry, I thought it was a meta so it wouldn't appear.  I'll fix